independent-publisher-connectors/Shopify Admin Content/apiProperties.json

```diff
@@ -0,0 +1,87 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "",
+          "scopes": [
+            "read_analytics",
+            "read_reports",
+            "write_orders",
+            "read_customers",
+            "write_customers",
+            "write_reports",
+            "read_content",
+            "write_content",
+            "read_themes",
+            "write_themes",
+            "read_product_listings",
+            "read_products",
+            "write_products",
+            "read_checkouts",
+            "write_checkouts",
+            "read_inventory",
+            "write_inventory",
+            "read_marketing_events",
+            "write_marketing_events",
+            "read_shopify_payments_disputes"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://yourstorename.myshopify.com/admin/oauth/authorize"
```

Comment:
@mamurshe Could you please help me review this PR - How this will gonna work, I didn't see any policy templates for custom connection parameters!!

Reply:
@sriyen-msft If you check the lines 74-81, there is a policy template to set the host-url. This policy template will set all the URLs.